src/main/java/hudson/plugins/jira/JiraCreateIssueNotifier.java

```diff
@@ -411,7 +408,21 @@ private void currentBuildResultSuccess(AbstractBuild<?, ?> build, BuildListener
         }
     }
 
-    private void progressWorkflowAction(AbstractBuild<?, ?> build, String issueId, Integer actionId) throws IOException {
+    static boolean isDone(Status status) {
+        if (status.getName().equalsIgnoreCase(finishedStatuses.Closed.toString()) ||
```

Comment:
why not only testing the category then?

Reply:
For backwards compatibility, somebody may have these statuses with different categories.
(Although it would not make sense).
If you are fine with requiring users to have the correct category on their statuses I can drop it.

Reply:
Ah, indeed, that makes sense, let's leave it for now.